src/components/UserProfile/TeamsAndProjects/AddTeamPopup.jsx

```diff
@@ -1,14 +1,27 @@
 import React, { useEffect, useState } from 'react';
 import { Button, Modal, ModalHeader, ModalBody, ModalFooter, Input, Alert } from 'reactstrap';
 import AddTeamsAutoComplete from './AddTeamsAutoComplete';
+import { useDispatch } from 'react-redux';
+import { postNewTeam, getAllUserTeams } from '../../../../src/actions/allTeamsAction';
 
 const AddTeamPopup = React.memo(props => {
+  const dispatch = useDispatch();
   const closePopup = () => {
     props.onClose();
   };
 
   const [selectedTeam, onSelectTeam] = useState(undefined);
   const [isValidTeam, onValidation] = useState(true);
+  const [isValidNewTeam, onNewTeamValidation] = useState(true);
```

Comment:
What is the point of this variable? Every time it is called its value is just getting modified, but is not used anywhere.